agent/local/testing.go

```diff
@@ -2,15 +2,21 @@ package local
 
 import (
 	"log"
-	"os"
 
 	"github.com/hashicorp/consul/agent/token"
+	"github.com/hashicorp/go-hclog"
 	"github.com/mitchellh/go-testing-interface"
 )
 
 // TestState returns a configured *State for testing.
 func TestState(t testing.T) *State {
-	result := NewState(Config{}, log.New(os.Stderr, "", log.LstdFlags), &token.Store{})
+	consulLogger := hclog.New(&hclog.LoggerOptions{
+		Level: log.LstdFlags,
```

Comment:
This should probably explicitly set the Output at least during this massive transition so everything is crystal clear, even if Stderr is the default.